app/models.py

```diff
@@ -26,3 +50,38 @@ def __repr__(self):
     def html_joke(self):
         """from (string) joke --> to (array) joke divided by enters"""
         return self.joke.split('\n')
+
+    def reactions_num(self, current_enum):
```

Comment:
I'd rename 'current_enum' to 'reaction_type' to keep the naming consistent. The argument's name should reflect what it is, enum is too general